BusyList/Handlers/DoneHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusyList.Commands;
+using System;
+
+namespace BusyList.Handlers
+{
+    public class DoneHandler : IHandler<DoneCommand>
```

Comment:
Looks like you based this branch on DoneHandler instead of Master. You'll need to rebase it, I'm afraid.

Reply:
Or set it to blocked by #20, that works too - as you need the TaskStatus property.

Reply:
I based it on master and how I can block it?